helm/reana/values.yaml

```diff
@@ -66,17 +66,17 @@ components:
       threads: 4
   reana_workflow_controller:
     imagePullPolicy: IfNotPresent
-    image: reanahub/reana-workflow-controller:0.7.1
+    image: reanahub/reana-workflow-controller:0.7.2
```

Comment:
Please update also `scripts/prefetch-images.sh` for researcher-oriented installation